Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml.cs

```diff
@@ -70,15 +73,51 @@ public sealed partial class SystemInformationPage : Page
         // To get the time the app was previously launched, not including this instance
         public string LastLaunchTime => SystemInformation.LastLaunchTime.ToString(Culture.DateTimeFormat);
 
-        // To get the number of times the app has been launched.
+        // To get the time the launch count was reset, not including this instance
+        public string LastResetTime => SystemInformation.LastResetTime.ToString(Culture.DateTimeFormat);
+
+        public string LastResetTimeProp { get; set; }
+
+        // To get the number of times the app has been launched sicne the last reset.
         public long LaunchCount => SystemInformation.LaunchCount;
 
+        public long LaunchCountProp { get; set; }
```

Comment:
Remove these extra properties and RaisePropertyChanged on the other properties
`RaisePropertyChanged(nameof(LaunchCount ));`